tests/data/placebo/test_ecs_service_taggable/ecs.ListServices_1.json

```diff
@@ -0,0 +1,17 @@
+{
+    "status_code": 200,
+    "data": {
+        "serviceArns": [],
+        "ResponseMetadata": {
```

Comment:
four of these ecs.ListServices api calls are redundant

Reply:
removed